README.rst

```diff
@@ -45,6 +45,10 @@ improvements:
 - Auto-imports submodules.
 - ``pip``-installable.
 
+To see auto imports from the current session: ``%autoimport -h``
```

Comment:
should this be `-l/--list` instead?  the problem of `-h` is that I expect people to just try this for getting --help (even though this is not implemented, which is fine).